openapi/db/path.py

```diff
@@ -244,3 +255,14 @@ def default_filter_field(self, field, op, value):
                 return field < value
             elif op == 'le':
                 return field <= value
+
+    def handle_unique_violation(self, exception):
+        match = re.match(unique_regex, exception.detail)
+        if not match:
+            raise exception
+
+        column = match.group('column')
+        value = match.group('value')
+        data = {column: 'already exists'}
```

Comment:
is `column` a valid field? We need to check.
For now, I would only pass the message